doc/source/setup-jupyterhub.rst

```diff
@@ -40,11 +40,22 @@ For the following steps, use your favorite code editor. We'll use the
    .. code-block:: yaml
 
       proxy:
-        secretToken: "RANDOM_STRING"
+        secretToken: "<OUTPUT-OF-`openssl rand -hex 32`>"
 
 .. Don't put an example here! People will just copy paste that & that's a security issue.
 
-4. Save the ``config.yaml`` file. If using the nano editor, hit ``Ctrl-X`` and
+4. **Azure AKS only** If you're on Microsoft Azure AKS, you must disable
```

Comment:
This feels very hacky, but I guess there isn't really a clear way around it. Hopefully AKS will get a little bit more mature soon and we can get rid of it.

Reply:
Yeah I'm assuming we'll re-write this as the process changes